src/cluster_migrateslots.c

```diff
@@ -353,6 +358,138 @@ void fireModuleSlotMigrationEvent(slotMigrationJob *job, int subevent) {
     zfree(info.slot_ranges);
 }
 
+sds clusterEncodeSlotImportsAuxField(int rdbflags) {
```

Comment:
Include RDB in this function name too? Or this function is internal? If it's internal, the name doesn't have to be that verbose, but then use `static` to mark it as internal.

```suggestion
static sds clusterEncodeSlotImportsAuxField(int rdbflags) {
```

Reply:
I did both :)